src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1973,9 +1967,7 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
                 add_link_lib_path(vec![llvm_libdir.trim().into()], &mut cmd);
             }
 
-            if !builder.config.dry_run()
-                && (matches!(suite, "run-make" | "run-make-fulldeps") || mode == "coverage-run")
-            {
+            if !builder.config.dry_run() && matches!(mode, "run-make" | "coverage-run") {
```

Comment:
This now checks the `mode` for `run-make`, instead of the `suite`. Is it the same?

Reply:
I think this is okay, because the test mode is "run-make" but the test suite name is also "run-make".